utam-core/src/test/java/utam/core/selenium/appium/MobileDriverAdapterTests.java

```diff
@@ -64,14 +66,14 @@ public void testCreation() {
   @Test
   public void testIsNative() {
     MockUtilities mock = new MockUtilities(AppiumDriver.class);
-    when(mock.getAppiumDriverMock().getContext()).thenReturn(NATIVE_CONTEXT_HANDLE);
+    when(((SupportsContextSwitching)mock.getAppiumDriverMock()).getContext()).thenReturn(NATIVE_CONTEXT_HANDLE);
```

Comment:
i think we need method mock.getSupportsContextSwitching in MockUtilities

Reply:
I don't think so. because in the constructor of `MockUtilities`, if we're using a mobile mock, we're adding that interface to the list of extra interfaces mocked by the mock (see https://github.com/salesforce/utam-java/pull/142/files#diff-a33d36772af6a57d13f2676916d645415db917ab92623d5b8295888419054dabL61)